docs/core-concepts/programs.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Programs
+---
+
+# Programs
+
+#### Takeout
+- Programs are STATELESS
```

Comment:
Instead of CAPS, maybe say `Programs are stateless and store state in separate accounts`.